internal/transport/transport.go

```diff
@@ -287,8 +287,9 @@ type Stream struct {
 	// On server-side it is unused.
 	status *status.Status
 
-	bytesReceived uint32 // indicates whether any bytes have been received on this stream
-	unprocessed   uint32 // set if the server sends a refused stream or GOAWAY including this stream
+	bytesReceived    uint32 // indicates whether any bytes have been received on this stream
+	unprocessed      uint32 // set if the server sends a refused stream or GOAWAY including this stream
+	headerWireLength int
```

Comment:
This should be grouped with other header stuff probably. And it needs a comment that it's only set by the server.

Reply:
Done.